xapi-profiles-structure.md

```diff
@@ -93,6 +93,7 @@ Property | Type | Description | Required
 `@context` | URI | SHOULD be `https://w3id.org/xapi/profiles/context` and MUST contain this URI if array-valued. | Required
 `type` | String | MUST be `Profile`. | Required
 `conformsTo` | URI | Canonical URI of the Profile specification version conformed to. The Profile specification version of this document is https://w3id.org/xapi/profiles#1.0. | Required
+`xAPIVersion` | Array | The version of xAPI this profile adheres to. When not provided, the Profile MUST adhere to the most recent version of xAPI | Optional
```

Comment:
*The version* -> *The minimum version*

Reply:
I guess, that we want to make the property Optional to be backwards compatible, but way may suggest a default value instead i ? (e.g. 2.0).

Reply:
make this explicit (2.0) and then add language that this is the default and can be changed based on the needs of the profile author.

When future versions come out, a profile that ref set 2.0 as xAPIVersion. 2.0 may be valid 2.1 but 2.1 may not be valid 2.0

add meaning of "minimum"